docs/serving/using-an-ssl-cert.md

```diff
@@ -1,140 +1,217 @@
 ---
```

Comment:
Given that SSL is deprecated, I think we should rename this file (and update all the corresponding links) -> using-an-tls-cert.md

I can add a corresponding redirect in knative.dev to redirect links from the old filename.

Reply:
renamed it to using-a-tls-cert.md